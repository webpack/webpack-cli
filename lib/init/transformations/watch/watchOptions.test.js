"use strict";

const defineTest = require("../../../utils/defineTest");

defineTest(
	__dirname,
	"watchOptions",
	"watch-0",
	{
		aggregateTimeout: 300,
		poll: 1000,
		ignored: "/node_modules/"
	},
	"init"
);

defineTest(
	__dirname,
	"watchOptions",
	"watch-1",
	{
		aggregateTimeout: 300,
		poll: 1000,
		ignored: "/node_modules/"
	},
	"init"
);

defineTest(
	__dirname,
	"watchOptions",
	"watch-2",
	{
		aggregateTimeout: 300,
		poll: 1000,
		ignored: "/node_modules/"
	},
	"init"
);
