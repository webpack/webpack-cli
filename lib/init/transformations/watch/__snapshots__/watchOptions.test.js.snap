// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`watchOptions transforms correctly using "watch-0" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    watchOptions: {
        aggregateTimeout: 300,
        poll: 1000,
        ignored: /node_modules/
    }
}
"
`;

exports[`watchOptions transforms correctly using "watch-1" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    watchOptions: {
        aggregateTimeout: 300,
        poll: 1000,
        ignored: /node_modules/
    }
}
"
`;

exports[`watchOptions transforms correctly using "watch-2" data 1`] = `
"module.exports = {
    entry: 'index.js',

    output: {
		filename: 'bundle.js'
	},

    watchOptions: {
        aggregateTimeout: 300,
        poll: 1000,
        ignored: /node_modules/
    }
}
"
`;
